sdk/tests/conformance2/rendering/blitframebuffer-multisampled-readbuffer.html

```diff
@@ -126,35 +93,13 @@
 }
 
 function checkPixel(color, expectedColor) {
-  var tolerance = 7;
+  var tolerance = 1;
```

Comment:
On my MacBook, it can pass even the tolerance is 0. Because we draw the rect with a solid color, so the only pixel diff comes from precision lost during srgb conversion, if it can do correct srgb conversion. precision lost would be very very small color diff.
 
However, set it to 7 or even more bigger, it can catch the error too. Because the diff would be very big if srgb conversion itself is wrong or not done.

I set the tolerance to 3 in the new commit. Please take another look.   @zhenyao and @kenrussell . (Zhenyao, welcome back to office, :) )